src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -28,10 +28,20 @@ const professionalClient = (() => {
         }
 
         const status = State.getResponse('get_account_status.status') || [];
-        if (is_in_page && /professional/.test(status)) {
+        if (is_in_page && /^professional$/.test(status)) {
```

Comment:
also doesn't work, if status is an array with multiple items. you can fake it in console and just try it. use an array like ['age_verified', 'professional'] and try your regex.

Reply:
because you are using ^ and $, it won't match. use something else to test if an item is exactly inside an array. there are multiple ways.

Reply:
@negarn why was regex used in the old code if that was the case?

Reply:
without ^ and $ it works, but it catches anything that has the word professional in it, such as professional_rejected, which is what caused the issue in the first place.

Reply:
I see. That makes sense.